ionit

```diff
@@ -209,7 +211,7 @@ def main(argv):
         "-t",
         "--templates",
         action="append",
-        help="Directory to search for Jinja templates " f"(default: {DEFAULT_TEMPLATES})",
+        help=f"Directory to search for Jinja templates (default: {str(DEFAULT_TEMPLATES)[2:-2]})",
```

Comment:
That string conversion looks ugly. So better dropping the idea of `DEFAULT_TEMPLATES` being a list and rename `DEFAULT_TEMPLATES` to `DEFAULT_TEMPLATE` to just contain `"etc"`.

Reply:
Or maybe for consistency: `DEFAULT_TEMPLATES_DIRECTORY`.